client/src/templates/Challenges/classic/DesktopLayout.js

```diff
@@ -84,17 +84,13 @@ class DesktopLayout extends Component {
           )}
 
           <ReflexElement flex={1} {...resizeProps}>
+            {challengeFile && showUpcomingChanges && !hasEditableBoundries && (
+              <EditorTabs />
+            )}
             {challengeFile && (
               <ReflexContainer key={challengeFile.key} orientation='horizontal'>
                 <ReflexElement flex={1} {...reflexProps} {...resizeProps}>
-                  {
-                    <Fragment>
-                      {showUpcomingChanges && !hasEditableBoundries && (
-                        <EditorTabs />
-                      )}
-                      {editor}
-                    </Fragment>
-                  }
+                  {<Fragment>{editor}</Fragment>}
```

Comment:
I just realised this 😅 , those curly braces around `Fragment` is not required.

Reply:
Oops, I didn't spot that either. They don't actually do anything in this case, right?

Reply:
I'll change that in my next pr